Brixcer_Box/platformio.ini

```diff
@@ -12,3 +12,13 @@
 platform = atmelavr
 board = 32u416m
 framework = arduino
+
+; [env:disco_l475vg_iot01a]
+; platform = atmelavr
+; board = uno
+; framework = arduino
+
+; [env:teensy31]
+; platform = teensy
+; board = teensy31
+; framework = arduino
```

Comment:
Do we want to commit this?

Reply:
Doesn't really matter. I removed it to see if GH will ignore the merge conflict (since you added the UNO to the list in one of your PRs.

Reply:
It did not... I fixed the merge anyway